src/demo/app/geo/geochart-demo.component.css

```diff
@@ -0,0 +1,313 @@
+:host {
```

Comment:
No reason to review any of this geochart demo stuff. It will all be removed when it gets implemented for realsies in metrics.